packages/react-native/src/primitives/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Text, View } from 'react-native';
+
+import { icons } from '../../assets';
+import { Icon } from '../Icon';
+import { IconButton } from '../IconButton';
+
+import { ErrorMessageProps } from './types';
+import { styles } from './styles';
+
+export default function Label({
+  accessibilityRole = 'alert',
+  children,
+  onDismiss,
+  style,
+  textStyle,
+  ...rest
+}: ErrorMessageProps): JSX.Element | null {
+  const [dismissed, setDismissed] = useState<boolean>(false);
```

Comment:
Few things here:
- don't think we need to track whether or not the `ErrorMessage` has been dismissed internally
- should not need to return `null`, if there is not need to render the component the parent should just not render it
- no need to pass `boolean` to the `useState` generic, TS will infer from the default argument